### ISSUE TITLE
Breakpoints only worked inside the _build/default/ directory

### ISSUE THREAD

**Issue by Mng12345** [hackwaly/ocamlearlybird#58]
The breakpoint can not be add in the source file, when i click the line in source file, the breakpoint is added in the file located in _build/default directory.
Screen-shot is here.
![image](https://github.com/hackwaly/ocamlearlybird/assets/21278900/0509aafc-e6ac-43a8-8e2c-97ea21f49a88)

**Comment by sim642** [comment]
I've found this to be more annoying in terms of modifying the files because after debugging, you might have the `_build` copy open instead of the original source, but that's read-only.

I think the problem is that dune-built artifacts are all based on the sources copied to `_build` and thus debug information refers to those paths instead of the ones in original sources. It's not as simple as just dropping `_build/default/` for various reasons:
1. Users might use some other dune contexts as `default`.
2. Some sources are generated, so no original source exists (unless dune promotion is being used).
3. Some sources are transformed (e.g. by ppx), so original and built sources are different (with different line numbers).

**Comment by Mng12345** [comment]
Yes, it did be annoying...... I have got notified a lot of times by warning the file in _build/default can not be modified.

**Comment by Mng12345** [comment]
![mnggiflab-video-to-gif (28)](https://github.com/hackwaly/ocamlearlybird/assets/21278900/11c20053-57c0-4c77-9db4-d6d493193134)

**Comment by acekingke** [comment]
I have the same problems
